src/main/scala/mesosphere/marathon/state/RunSpec.scala

```diff
@@ -31,6 +35,12 @@ trait RunnableSpec extends plugin.RunSpec {
   def gpus: Int
 }
 
+object RunnableSpec {
+  // TODO (pods): These can be removed when AppDefinition/RunSpec usages have been replaced with RunnableSpec
+  implicit def runnableSpecToAppDefinition(spec: RunnableSpec): AppDefinition = spec.asInstanceOf[AppDefinition]
```

Comment:
I removed this completely in the branch and favored "let's pattern match with TODOs" until we can actually remove the knowledge. This has the benefit that marathon works.